code/game/objects/items/tools/weldingtool.dm

```diff
@@ -385,6 +385,16 @@
 	icon_state = "clockwelder"
 	item_state = "brasswelder"
 
+/obj/item/weldingtool/experimental/ashwalker
+	name = "necropolis welding torch"
+	desc = "A mysterious welding tool with its origins in the depths of the necropolis. A mysterious force keeps replenishing its."
```

Comment:
Doesn't seem right, looks like a typo at the end.

Reply:
Maybe the welding tool could instead be lava-based, and need interaction with lava to regain the fuel?

Reply:
I mean, they do start with an experimental welding tool currently, which I intended to replace with this lesser version.